src/filesystem/pagination/interface/api/PageType.h

```diff
@@ -3,7 +3,7 @@
 
 enum PageType {
 
-  BP_TREE_HEADER_PAGE,
+  BP_TREE_HEADER_PAGE = 0,
```

Comment:
Why do we explicitly set it to `0`?

Reply:
I was just being paranoid, I don't think you have to though